environments/se/core/src/main/java/org/jboss/weld/environment/se/DefaultExceptionHandler.java

```diff
@@ -0,0 +1,21 @@
+package org.jboss.weld.environment.se;
+
+import java.util.stream.Stream;
+
+import org.jboss.logging.Logger;
+
+public class DefaultExceptionHandler implements ExceptionHandler {
+
+    private static final Logger logger = Logger.getLogger(DefaultExceptionHandler.class);
+
+    @Override
+    public Stream<Class<? extends Throwable>> getExceptionTypes() {
+        return null;
```

Comment:
I think this would result in NPE (see also the default `supports()` implementation).

Reply:
The ``DefaultExceptionHandler`` has to be at the end of the list and isn't registered via the ``ServiceLoader``.  See the ``orElse`` clause in ``StartMain``.